src/open_inwoner/kvk/branches.py

```diff
@@ -3,7 +3,3 @@
 
 def kvk_branch_selected_done(session) -> bool:
     return KVK_BRANCH_SESSION_VARIABLE in session
```

Comment:
Do we still  need this? I can't find any references to `kvk_branch_selected_done`, and the only refs to `KVK_BRANCH_SESSION_VARIABLE` are in tests.

Reply:
Discussed: we'll merge this one because other PR's need the changes. Cleanup will be done on develop